src/content/blog/2023-09-04-tus-node-server-v1.0.0.md

```diff
@@ -0,0 +1,172 @@
+---
+title: 'Tus Node.js 1.0.0: new packages, rewritten in TypeScript, and much more'
+author: murderlon
+date: 2023-09-04
+---
+
+The [Node.js tus server](https://github.com/tus/tus-node-server) has been
+revived and is now officially semver stable! It has been completely rewritten in
+TypeScript, split into separate packages, has new events and hooks system,
+elaborate docs with examples, and many fixes and features we’ll dive into below.
+
+## History
+
+The Node.js tus server, [tus-node-server][], was originally published by
+[@bhstahl][] in May 2016. As with many open source projects, not everyone has
+the time to maintain things for free in their own time and as such
+[tus-node-server][] was more or less abandoned at the end of 2018.
```

Comment:
I don't remember the exact details, but Ben Stahl didn't work on tus-node-server in his free time. At the time, he worked for Vimeo, so I assume that his involvement was work-related. He left Vimeo at the middle of 2018 and then tus-node-server was left abandoned: https://linkedin.com/in/bhstahl But I am not a 100% sure about this.

Reply:
Perhaps it doesn't matter, the reason for abandoning it seemed to be life got in the way, how he worked on it during the maintenance period is not explicitly stated. Unless you feel it's important to attribute Vimeo?

Reply:
I think it was Soundcloud actually where he worked when he did most of the tus-node-server work?

Reply:
But yeah, maybe not relevant where he worked, employers never paid for him working on the project afaik anyway